packages/curve-ui-kit/src/features/connect-wallet/lib/init.ts

```diff
@@ -1,69 +0,0 @@
-import { ThemeKey } from '@ui-kit/themes/basic-theme'
-import * as onboard from '@web3-onboard/react'
-import {
-  bitget,
-  coinbaseWalletSdk,
-  fortmatic,
-  frontier,
-  gnosis,
-  injected,
-  ledger,
-  metamaskSDKWallet,
-  phantom,
-  torus,
-  trezor,
-  trust,
-  walletConnect,
-} from './utils/walletModules'
-
-export const initOnboard = (
```

Comment:
can we remove the dependencies :pray: ?
also this line https://github.com/curvefi/curve-frontend/blob/f9f9e5b726ddaf8cd6a45b74f6a8ff8effec63de/tests/cypress/support/e2e.ts#L10